packages/i18n/I18nDecorator/I18nDecorator.js

```diff
@@ -104,8 +103,7 @@ const I18nDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			locale: PropTypes.string
 		}
 
-		static contextTypes = stateContextTypes
-		static childContextTypes = {...contextTypes, ...stateContextTypes}
+		static childContextTypes = {...contextTypes}
```

Comment:
No need to clone the object here. Pass by ref is okay now that it's only 1 `contextTypes`.